relays/bin-substrate/src/rialto_millau/cli.rs

```diff
@@ -382,47 +291,11 @@ pub enum ToRialtoMessage {
 		recipient: AccountId,
 		/// Amount of target tokens to send in target chain base currency units.
 		#[structopt(long)]
-		amount: bp_rialto::Balance,
-	},
-	/// A call to the Millau Bridge Messages pallet to send a message over the bridge.
-	MillauSendMessage {
-		/// Hex-encoded lane id that should be served by the relay. Defaults to `00000000`.
-		#[structopt(long, default_value = "00000000")]
-		lane: HexLaneId,
-		/// Raw SCALE-encoded Message Payload to submit to the messages pallet.
-		#[structopt(long)]
-		payload: HexBytes,
-		/// Declared delivery and dispatch fee in base source-chain currency units.
-		#[structopt(long)]
-		fee: bp_rialto::Balance,
-	},
-}
-
-/// All possible messages that may be delivered to the Millau chain.
-#[derive(StructOpt, Debug)]
-pub enum ToMillauMessage {
-	/// Raw bytes for the message
-	Raw {
-		/// Raw, SCALE-encoded message
-		data: HexBytes,
-	},
-	/// Make an on-chain remark (comment).
-	Remark {
-		/// Size of the remark. If not passed, small UTF8-encoded string is generated by relay as remark.
-		#[structopt(long)]
-		remark_size: Option<ExplicitOrMaximal<usize>>,
-	},
-	/// Transfer the specified `amount` of native tokens to a particular `recipient`.
-	Transfer {
-		/// SS58 encoded account that will receive the transfer (must have SS58Prefix = 42)
-		#[structopt(long)]
-		recipient: AccountId,
-		/// Amount of target tokens to send in target chain base currency units.
-		#[structopt(long)]
-		amount: bp_millau::Balance,
+		amount: Balance,
 	},
-	/// A call to the Rialto Bridge Messages pallet to send a message over the bridge.
-	RialtoSendMessage {
+	// TODO [#853] Support multiple bridges.
+	/// A call to the specific Message Lane pallet to send a message over the bridge.
```

Comment:
Quote from #853:

```
Currently the Call CLI enum, supports BridgeSendMessage variant. However in case there are multiple bridges on one source chain, we need to specify what Target chain we want to send the message to.
```

I'm a bit confused now - I thought that this variant is to call over the bridge the other bridge (deployed on the target chain) to send other message. So it may be `Source --- (M1) ---> Target --- (M2) ---> Source` or `Source1 --- (M1) ---> Target --- (M2) ---> Source2` iiuc. So the quote should be the "multiple bridges on the target chain", no? 

I guess it was introduced in the PR about printing encoded messages, so it isn't actually used in messages sending. In practice it is only used by the `encode-call` subcommand. But since it is available to the `send-message` command as well, maybe it worth at least to expand docs a bit - e.g instead of "specific Message Lane pallet" say "Message Lane pallet on the target chain", for payload change it to the "encoded Call of the source chain", etc.

P.S.: 'Message Lane pallet' is also changed to 'Messages pallet' everywhere ;)

Reply:
Yes, that's a good point. Indeed I was thinking about it from the `encode-call` perspective, I'll update the docs.